test/factories/BattleFactory.js

```diff
@@ -0,0 +1,18 @@
+const { faker, models } = require('test/testHelper');
```

Comment:
why you require all models? You have Battle at testHelper exports

Reply:
done